src/lib/utils/notifications/index.ts

```diff
@@ -136,9 +136,39 @@ const sendNotification = async (
     })),
   });
 
+  // If no fields in database were affected, it failed
   if (databaseResult.count < 1) {
     throw error(500, `Failed to create notifications`);
   }
+
+  // Get user's expo tokens and use them to send push notifications
+  const pushNotificationMembers = receivingMembers
+    .filter(
+      (
+        member, // Filter all members who don't have push notification enabled for this type of notifications
+      ) =>
+        member.subscriptionSettings.some(
+          (settings) => settings.pushNotification,
+        ),
+    )
+    .map((member) => member.id); // Return an array of strings with their memberIds
+  const expoTokens = await prisma.expoToken.findMany({
+    where: {
+      memberId: {
+        in: pushNotificationMembers,
+      },
+    },
+  });
+
+  if (expoTokens != undefined) {
+    sendPushNotifications(
+      expoTokens.map((token) => token.expoToken),
+      title,
+      message,
+      type,
+      link,
+    );
+  }
```

Comment:
should be extracted to a new function. I think the same goes for the `databaseResult` query above, so basically we would have two async functions:
- `sendInAppNotifications(memberIds[], title...)`
- `sendPushNotifications(memberIds[], title,...)`

These could then be called and awaited simulatenously, reducing the time of this function. I think that's crucial because it's going to be run often and sometimes having to send a lot of notifications at once (like some news tags). I think that's how it was done before and is why it's called `databaseResult` (because it awaited a `Promise.allSettled([...])` call